components/pages/workPageLayout/WorkPageButtons.tsx

```diff
@@ -56,6 +55,20 @@ const WorkPageButtons = ({ workId }: WorkPageButtonsProps) => {
           </Button>
         </>
       )}
+      {isPodcast(selectedManifestation) && (
```

Comment:
I think I like having them separated. This is because at some point they might introduce another material type or condition to consider. Also it is more readable in this way IMO. But I understand your point, I was a bit back and forward on this one.